lms/templates/dashboard/_dashboard_credit_info.html

```diff
@@ -17,69 +20,53 @@
             )}
         </p>
     % if not credit_status["purchased"] and not credit_status["error"] :
-        <p class="credit-eligibility-msg">
-            % if credit_status["deadline"] < datetime.datetime.now(pytz.UTC) + datetime.timedelta(days=30):
-                ${_("The opportunity to purchase credit for this course expires on {deadline}. You've worked hard - don't miss out!").format(deadline=get_default_time_display(credit_status["deadline"]))}
-            % else:
-                ${_("Congratulations - you have met the requirements for credit in this course!")}
-            % endif
+        <p class="message-copy credit-eligibility-msg">
+            ${_("You are now eligible for credit. {congrats}").format(
+                    congrats="<b>Congratulations</b>",
+                )
+            }
         </p>
         <div class="purchase_credit">
-            <a class="btn credit-btn purchase-credit-btn" href="${settings.ECOMMERCE_PUBLIC_URL_ROOT}/credit/checkout/${credit_status['course_key']}" target="_blank" data-course-key="${credit_status['course_key']}">${_("Purchase Course Credit")}</a>
+            <a class="btn credit-btn purchase-credit-btn" href="${settings.ECOMMERCE_PUBLIC_URL_ROOT}/credit/checkout/${credit_status['course_key']}" target="_blank" data-course-key="${credit_status['course_key']}">${_("Get credit")}</a>
         </div>
     % elif credit_status["request_status"] in [None, "pending"] and not credit_status["error"] :
         % if credit_status["request_status"] == "pending":
             <p class="message-copy credit-request-pending-msg">
-            ${_("Thanks for your payment! We are currently processing your course credit. You'll see a message here when the transaction is complete. For more information, see {provider_link}.").format(
-                    provider_link=u'<a href="{provider_url}">{provider_name}</a>'.format(
-                        provider_url=credit_status["provider_status_url"],
-                        provider_name=credit_status["provider_name"],
+                ${_("Thank you. Your credit is processing. Please see {link} for more information.").format(
```

Comment:
This does not work for internationalization. Who is here in Cambridge that can help me figure out how to move forward.